addons/with-web/index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = (config = {}) => {
+    const {
+        transform = {},
+        collectCoverageFrom = [],
+        coveragePathIgnorePatterns = [],
+        setupFilesAfterEnv = [],
+    } = config;
+
+    config.transform = {
+        '\\.data-url\\.': require.resolve('./data-url'),
+        '\\.inline\\.': require.resolve('./inline'),
+        ...transform,
+    };
+
+    config.collectCoverageFrom = [
+        ...collectCoverageFrom,
+        'www/**/*.js',
+    ];
+
+    config.coveragePathIgnorePatterns = [
+        ...coveragePathIgnorePatterns,
+        '.*\\.data\\.js',
+    ];
+
+    config.setupFilesAfterEnv = [
+        ...setupFilesAfterEnv,
+        '@testing-library/jest-dom/extend-expect',
```

Comment:
Something to consider later is that we may use RTL stuff like react packages, such as react-promiseful. This means that a addon just for RTL or react would be nice.